website/pages/en/billing.mdx

```diff
@@ -194,3 +194,16 @@ Of course, both new and existing users can reach out to Edge & Node's BD team fo
 ### Can I withdrawal GRT from my billing balance?
 
 Yes, you can always withdrawal GRT that has not already been used for queries from your billing balance. The billing contract is only designed to bridge GRT from Ethereum mainnet to the Arbitrum network. If you'd like to transfer your GRT from Arbitrum back to Ethereum mainnet, you'll need to use the [Arbitrum Bridge](https://bridge.arbitrum.io/?l2ChainId=42161).
+
+### What happens when GRT is depleted? Will I get a warning?
+
+Users will receive an email when GRT is close to depletion. You will receive several notifications before billing runs out.
+
+### Which chain does GRT need to be on?
+
+Albitrum is the best option, but users can bridge their GRT and ETH using any of the following options:
```

Comment:
```suggestion
Arbitrum is the best option, but users can bridge their GRT and ETH using any of the following options:
```
Just fixing the typo, but I actually think this whole question and answer is a bit confusing. There's just no way in the UI to add GRT to your balance on any chain other than Arbitrum, no?

Reply:
Let me double check again.